package.json

```diff
@@ -16,7 +16,9 @@
   "dependencies": {
     "@types/jasmine": "^2.6.0",
     "@types/node": "^8.0.46",
+    "@types/yargs": "^8.0.2",
     "ts-simple-ast": "^0.60.0",
-    "typescript": "2.5.2"
+    "typescript": "^2.5.2",
```

Comment:
The `ts-simple-ast` module version `0.60.0` requires the exact typescript version `2.5.2`. Remove the `^`.